scripts/packages/chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -50,7 +50,7 @@ if ($packageParameters)
 }
 Install-ChocolateyPath -PathToInstall "$msys2Path\usr\bin" -PathType "Machine"
 
-$addToMsysPath = ($packageDir -replace 'c:\\','/c/') -replace '\\','/'
+$addToMsysPath = ($packageDir -replace '^([a-zA-Z]):\\(.*)','/$1/$2') -replace '\\','/'
```

Comment:
That's true. I like this pattern more since it's more explicit, but I don't feel strongly.